scripts/oneoff/fix_accidentially_published_g10_services.sql

```diff
@@ -0,0 +1,84 @@
+/*
+To apply this script run the below command
+Targe the correct environment with `cf target -s <desired environment>`
+`cf conduit digitalmarketplace_api_db -- psql < ./scripts/oneoff/fix_accidentially_published_g10_services.sql`
+*/
+
+START transaction;
+SELECT
+   services.service_id,
+   ds.id AS draft_id,
+   services.supplier_id INTO TEMP TABLE naughty_services
+FROM
+   services
+   LEFT JOIN
+      draft_services AS ds
+      ON services.service_id = ds.service_id
+   LEFT JOIN
+      supplier_frameworks AS sf
+      ON services.supplier_id = sf.supplier_id
+WHERE
+   services.framework_id = (SELECT id FROM frameworks WHERE slug = 'g-cloud-10')
+   AND sf.on_framework = 'f'
+   AND sf.framework_id = (SELECT id FROM frameworks WHERE slug = 'g-cloud-10')
+;
+
+UPDATE
+   draft_services
+SET
+   service_id = NULL
+WHERE
+   framework_id = (SELECT id FROM frameworks WHERE slug = 'g-cloud-10')
+   AND supplier_id IN (SELECT supplier_id FROM naughty_services)
```

Comment:
It'd be more consistent to test `service_id` here rather than `supplier_id`?

Reply:
I think i've made it even simpler than than! Let me know if you think its ok.